README.md

```diff
@@ -5,60 +5,56 @@ Jupiter Interface
+Additionally a `Test` dependency to the runtime library `jupiter-interface` is required for every module which wants to run JUnit 5 tests:
 
-### Integration Testing
+```scala
+libraryDependencies ++= Seq(
+  "com.github.sbt.junit" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % Test
+)
+```
 
-Basic configuration of a project to have `it:test` is [documented in SBT](https://www.scala-sbt.org/0.13/docs/Testing.html#Integration+Tests). For `Jupiter Interface` to work, you will also need to install plugin specific settings and tasks to the `it` configuration, as follows:
+Note: If you want to restore default behaviour like in versions before `0.8.0` you can globally activate this plugin by adding the runtime dependency to `ThisBuild / libraryDependencies`:
 
+```scala
+ThisBuild / libraryDependencies += "com.github.sbt.junit" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % Test
 ```
-lazy val root = project.in(file("."))
-  .configs(IntegrationTest)
-  .settings(Defaults.itSettings: _*)
-  .settings(inConfig(IntegrationTest)(JupiterPlugin.scopedSettings): _*)
-  ...
-```
-Don’t forget to mark test dependencies as `test,it` if you have both unit and integration tests.
 
 ## Releases
 
-**Note** that unfortunately artifacts which were hosted at `Bintray` are no longer available since
-the service has been discontinued. This includes releases up to and including `0.8.4`. Starting with
-`0.9.0` artifacts are published on `Maven Central` through `Sonatype`.
-
-Since version `0.6.0` this plugin is cross built for `scala_2.10/sbt_0.13` and `scala_2.12/sbt_1.0`.
-
- SBT Plugin      | JUnit Platform | JUnit Engine
+ sbt Plugin      | JUnit Platform | JUnit Engine
 :----------------|:---------------|:-------------
- 0.1.x           | 1.0.0-M4       | 5.0.0-M4
- 0.2.x           | 1.0.0-M5       | 5.0.0-M5
- 0.3.x           | 1.0.0-M6       | 5.0.0-M6
- 0.4.x           | 1.0.0-RC2      | 5.0.0-RC2
- 0.5.x           | 1.0.0          | 5.0.0
- 0.6.x           | 1.0.0          | 5.0.0
- 0.7.x           | 1.1.0          | 5.1.0
- 0.8.x           | 1.1.x          | 5.1.x
- 0.9.x           | 1.7.x          | 5.7.x
- 0.10.x          | 1.8.x          | 5.8.x
  0.11.x          | 1.9.x          | 5.9.x
+ 0.10.x          | 1.8.x          | 5.8.x
+ 0.9.x           | 1.7.x          | 5.7.x
+ 0.8.x           | 1.1.x          | 5.1.x
+ 0.7.x           | 1.1.0          | 5.1.0
+ 0.6.x           | 1.0.0          | 5.0.0
+ 0.5.x           | 1.0.0          | 5.0.0
+ 0.4.x           | 1.0.0-RC2      | 5.0.0-RC2
+ 0.3.x           | 1.0.0-M6       | 5.0.0-M6
+ 0.2.x           | 1.0.0-M5       | 5.0.0-M5
+ 0.1.x           | 1.0.0-M4       | 5.0.0-M4
+
+**Note** `0.9.0`+ artifacts are published on Maven Central via Sonatype OSS. Previous versions were available via <https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/net.aichler/sbt-jupiter-interface/scala_2.10/sbt_0.13/>, whose underlying host has migrated from Bintray to Scala Center Artifactory instance.
```

Comment:
Replaced incorrect note

> **Note** that unfortunately artifacts which were hosted at `Bintray` are no longer available since
the service has been discontinued.

with a link to Arty.